src/core/Button/index.tsx

```diff
@@ -1,25 +1,19 @@
 import { ButtonProps as RawButtonProps } from "@material-ui/core";
 import React from "react";
 import {
+  ExtraProps,
   PrimaryMinimalButton,
   RoundedButton,
   SecondaryMinimalButton,
   SquareButton,
   StyledButton,
 } from "./style";
 
-interface SdsProps {
-  isAllCaps?: boolean;
-  isRounded?: boolean;
-  sdsStyle?: "minimal" | "rounded" | "square";
-  sdsType?: "primary" | "secondary";
-}
-export type ButtonProps = RawButtonProps & SdsProps;
+export type ButtonProps = RawButtonProps & ExtraProps;
 
 const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
   (props: ButtonProps, ref): JSX.Element | null => {
-    const sdsStyle = props?.sdsStyle;
-    const sdsType = props?.sdsType;
+    const { color = "primary", sdsStyle, sdsType } = props;
```

Comment:
since MUI Button already accepts `color` prop, we default the `color` selection to `"primary"` since SDS only uses the primary color most of the time